HISTORY.md

```diff
@@ -12,6 +12,9 @@
 ### Performance Improvements
 * On ARM platform, use `yield` instead of `wfe` to relax cpu to gain better performance.
 
+### Public API change
+* Update DB::StartIOTrace and remove Env object from the arguments as its redundant and DB already has Env object that is passed down to IOTracer::StartIOTrace
```

Comment:
The env is passed when constructing ```TraceWriter``` right?

Reply:
Yes. And since DB already has ```env_``` so that is passed down to TraceWriter.